frontend/packages/container-security/src/components/summary.tsx

```diff
@@ -130,14 +140,23 @@ export const SecurityBreakdownPopup: React.FC<SecurityBreakdownPopupProps> = ({
                         v.metadata.name
                       }`}
                     >
-                      {pluralize(
-                        resource.filter(({ metadata }) => metadata.name === v.metadata.name).length,
-                        'namespace',
-                      )}
+                      {namespace
+                        ? imageNameClamped(
+                            resource.find(({ metadata }) => metadata.name === v.metadata.name).spec
+                              .image,
+                          )
+                        : pluralize(
+                            resource.filter(({ metadata }) => metadata.name === v.metadata.name)
+                              .length,
+                            'namespace',
+                          )}
                     </Link>
                   </span>
                   <div className="text-secondary">
-                    <ExternalLink href={quayURLFor(v)} text={`${v.status?.fixableCount} fixable`} />
+                    <ExternalLink
+                      href={quayURLFor(v)}
+                      text={`${getVulnerabilityCountText(v, namespace)} fixable`}
```

Comment:
as you added util here m shouldn't we return `fixable` as well from util now

Reply:
done